app/controllers/metadata_presenter/answers_controller.rb

```diff
@@ -72,6 +80,12 @@ def upload_files
       @page_answers.page.upload_components.each do |component|
         answer = user_data[component.id]
 
+        original_filename = answer.nil? ? @page_answers.send(component.id)['original_filename'] : answer['original_filename']
+
+        if original_filename.presence
```

Comment:
```suggestion
        if original_filename.present?
```

Should we return a boolean rather than the `original_filename` object in this statement? Tbh I'm not sure if it amounts to the same thing

Reply:
pretty much does, O think present is probably better?